app/src/main/java/org/simple/clinic/medicalhistory/MedicalHistoryDiagnosisWithTreatment.kt

```diff
@@ -0,0 +1,115 @@
+package org.simple.clinic.medicalhistory
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import com.google.android.material.card.MaterialCardView
+import org.simple.clinic.databinding.ListMedicalhistoryDiagnosisWithTreatmentBinding
+
+class MedicalHistoryDiagnosisWithTreatment(
+    context: Context,
+    attrs: AttributeSet?
+) : MaterialCardView(context, attrs) {
+  private var binding: ListMedicalhistoryDiagnosisWithTreatmentBinding? = null
+
+  private val diagnosisTextView
+    get() = binding!!.diagnosisTextView
```

Comment:
No necessary to change. We can reassign `binding!!` to a variable here as well to avoid using nullable variables, no?